dll/win32/shdocvw/CFavBand.cpp

```diff
@@ -530,31 +530,38 @@ STDMETHODIMP CFavBand::Select(long paramC)
 
 // *** INamespaceProxy ***
 
-STDMETHODIMP CFavBand::GetNavigateTarget(long paramC, long param10, long param14)
+STDMETHODIMP CFavBand::GetNavigateTarget(
+    PCIDLIST_ABSOLUTE pidl,
+    PIDLIST_ABSOLUTE ppidlTarget,
+    PULONG pulAttrib)
 {
     UNIMPLEMENTED;
     return E_NOTIMPL;
 }
 
-STDMETHODIMP CFavBand::Invoke(long paramC)
+STDMETHODIMP CFavBand::Invoke(PCIDLIST_ABSOLUTE pidl)
 {
     UNIMPLEMENTED;
     return E_NOTIMPL;
 }
 
-STDMETHODIMP CFavBand::OnSelectionChanged(long paramC)
+STDMETHODIMP CFavBand::OnSelectionChanged(PCIDLIST_ABSOLUTE pidl)
 {
     UNIMPLEMENTED;
     return E_NOTIMPL;
 }
 
-STDMETHODIMP CFavBand::RefreshFlags(long paramC, long param10, long param14)
+STDMETHODIMP CFavBand::RefreshFlags(
+    PULONG pdwStyle,
+    PULONG pdwExStyle,
+    PULONG dwEnum)
 {
     UNIMPLEMENTED;
     return E_NOTIMPL;
 }
 
-STDMETHODIMP CFavBand::CacheItem(long paramC)
+STDMETHODIMP CFavBand::Reserved1(
```

Comment:
A comment here if you know the actual use of the function would be nice.

Reply:
I don't know.

Reply:
It is actually known (public headers):
https://github.com/tpn/winsdk-10/blob/9b69fd26ac0c7d0b83d378dba01080e93349c2ed/Include/10.0.16299.0/um/ienamespacecontrol.idl#L39

Reply:
Fixed.

Reply:
Comments added.